### PR TITLE
Fix: Misc minor design changes

### PR DESCRIPTION
**Pull request by rameshmane7218**
## Change type

- [x] fix: Tag color, should be grey when tab is inactive (EE)
- [x] fix: In OSS, count itself is not being displayed. needs a fix
- [x] fix: Wrong `New record` icon size leading to mis-alignment
- [x] fix: Enable public viewing & access, colour different, should be #101015
- [x] fix: Enable Public Viewing & Share Base text is not capitalize each word
- [x] fix: Toolbar dropdown menu border colour
- [x] fix: For rich text field (long text with rich text option enabled), display expand icon only on hover
- [x] fix: Rename View In Dropdown menu #7136 
- [x] fix: Expanded json editor modal z-index issue

## Additional information / screenshots (optional)
Please refer [getoutline docs](https://nocodb.getoutline.com/s/bf513045-e63f-4133-951a-211d78391235) for screenshots

### CODE REVIEW

**Comment by dstala** [comment]
- [ ] border missing for 
  - [ ] row height dropdown
  - [ ] dropdown's related to webhook (multiple)
  - [ ] field type in edit field modal
  - [ ] field type in new field modal
  - [ ] picker modals for date, date time, time, select fields
  - [ ] new data source : type, ssl mode, inflection

File: packages/nc-gui/components/project/View.vue [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <script lang="ts" setup>
 import { useTitle } from '@vueuse/core'
 import NcLayout from '~icons/nc-icons/layout'
-const { openedProject } = storeToRefs(useBases())
+const basesStore = useBases()
+const { getProjectUsers } = basesStore
+const { openedProject, activeProjectId, baseUserCount } = storeToRefs(basesStore)
 const { activeTables } = storeToRefs(useTablesStore())
 const { activeWorkspace, workspaceUserCount } = storeToRefs(useWorkspace())
+import { isEeUI } from '#imports'
```
Comment (wingkwong) [inline packages/nc-gui/components/project/View.vue:9]: put all import together. i.e. move this line after L3.

File: packages/nc-gui/components/project/View.vue [hunk 2]
```diff
@@ -1,9 +1,12 @@
 <script lang="ts" setup>
 import { useTitle } from '@vueuse/core'
 import NcLayout from '~icons/nc-icons/layout'
-const { openedProject } = storeToRefs(useBases())
+const basesStore = useBases()
+const { getProjectUsers } = basesStore
+const { openedProject, activeProjectId, baseUserCount } = storeToRefs(basesStore)
```
Comment (wingkwong) [inline packages/nc-gui/components/project/View.vue:6]: normally we add a line break in-between here

File: packages/nc-gui/components/dashboard/TreeView/ViewsNode.vue [hunk 3]
```diff
@@ -128,6 +128,17 @@ onKeyStroke('Enter', (event) => {
   }
 })
 
+const onRenameMenuClick = () => {
+  if (isMobileMode.value) return
+  if (!isUIAllowed('viewCreateOrEdit')) return
```
Comment (wingkwong) [inline packages/nc-gui/components/dashboard/TreeView/ViewsNode.vue:133]: combine together with `||`

File: packages/nc-gui/components/project/View.vue [hunk 4]
```diff
@@ -119,14 +148,14 @@ watch(
               <GeneralIcon icon="users" class="!h-3.5 !w-3.5" />
               <div>{{ $t('labels.members') }}</div>
               <div
-                v-if="workspaceUserCount"
+                v-if="isEeUI ? workspaceUserCount : baseUserCount"
```
Comment (wingkwong) [inline packages/nc-gui/components/project/View.vue:151]: L151 & L158 are same - define once and use multiple times instead.

File: packages/nc-gui/components/project/View.vue (hunk 1 above)
Comment (rameshmane7218) [reply packages/nc-gui/components/project/View.vue]: Done

File: packages/nc-gui/components/dashboard/TreeView/ViewsNode.vue (hunk 3 above)
Comment (rameshmane7218) [reply packages/nc-gui/components/dashboard/TreeView/ViewsNode.vue]: Fixed

File: packages/nc-gui/components/project/View.vue (hunk 4 above)
Comment (rameshmane7218) [reply packages/nc-gui/components/project/View.vue]: Updated

**Review by dstala — APPROVED** [review]

**Pull request merged**